examples/testing/multi_frameworks_toolchain/BUILD

```diff
@@ -0,0 +1,61 @@
+load("@io_bazel_rules_scala//scala:providers.bzl", "declare_deps_provider")
+load("@io_bazel_rules_scala//testing/toolchain:toolchain.bzl", "scala_testing_toolchain")
+
+scala_testing_toolchain(
+    name = "testing_toolchains_with_all_deps_impl",
+    dep_providers = [
+        ":junit_classpath_provider",
```

Comment:
Maybe reuse existing deps providers from rules_scala? Example would be smaller and focused.

Reply:
I'm actually thinking how to make providers private to rules_scala - they aren't meant for external reuse. Ideally this example should come with different loader like `rules_jvm_external` with different external names to fully demonstrate custom toolchain setup.